src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class Machine {
+    public void doWork() {
+        System.out.println("Machine started its work");
+    }
+    public void stopWork() {
+        System.out.println("Machine Stopped its work");
+    }

```

Comment:
according to task description these methods should be abstract without implementation:
"Create class Machine containing methods public abstract void doWork() and public abstract void stopWork()."